Scripts/data_display.gd

```diff
@@ -2,6 +2,7 @@ extends Control
 
 var data
 var elems
+var maxValue = 0
```

Comment:
Since GDScript is dynamically typed, for clarity, I recommend the default value be `null` instead of 0.